recipes/python-glmnet/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "v2.0.0" %}
+
+package:
+  name: glmnet
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/civisanalytics/python-glmnet.git
```

Comment:
I believe the preference is to get the source from pypi or the github release archive rather than a git tag:

http://conda-forge.github.io/docs/meta.html#build-from-tarballs-not-repos